core/domain/collection_domain.py

```diff
@@ -1266,15 +1414,15 @@ def is_editable_by(self, user_id):
             or self.community_owned
         )
 
-    def is_private(self):
+    def is_private(self) -> bool:
         """Checks whether the collection is private.
 
         Returns:
             bool. Whether the collection is private.
         """
-        return self.status == constants.ACTIVITY_STATUS_PRIVATE
+        return bool(self.status == constants.ACTIVITY_STATUS_PRIVATE)
```

Comment:
Why is the `bool` needed?

Reply:
`constants.ACTIVITY_STATUS_PRIVATE` is considere to be any type, so to make this bool type i added this

Reply:
Okay, explain it with a code comment please.

Reply:
Done!